strax/context.py

```diff
@@ -1844,30 +1844,89 @@ def get_meta(self, run_id, target) -> dict:
 
     get_metadata = get_meta
 
-    def compare_metadata(self, run_id, target, old_metadata):
+    def compare_metadata(self, data1, data2, return_results=False):
         """Compare the metadata between two strax data.
```

Comment:
Can you add to the doc string which explains the pintout of the comparison. I was a little confused which direction things are compared is red the first and green the second file?